docs/onenote/index.yml

```
@@ -64,7 +64,23 @@ landingContent:
             url: ../testing/test-debug-office-add-ins.md
           - text: Deploy and publish a OneNote add-in
             url: ../publish/publish.md
+            
+  # Card (optional)
+  - title: OneNote Developer Resources
```

Since very little here is OneNote-specific, can we leave this to match the "Resources" card found in the WXP landing pages?
```suggestion
  - title: Resources
```